include/merlin/core_kernels/lookup.cuh

```diff
@@ -21,6 +21,44 @@
 namespace nv {
 namespace merlin {
 
+template <typename K>
+struct FoundFunctorV1 {
+  __host__ __device__ FoundFunctorV1(bool* __restrict founds_)
+      : founds(founds_) {}
+
+  __forceinline__ __device__ void operator()(const int idx, const K /*key*/,
+                                         const bool found) {
```

Comment:
Please format the code by referring the [guidance](https://github.com/NVIDIA-Merlin/HierarchicalKV/blob/master/STYLE_GUIDE.md)

Reply:
fixed